tests/e2e/specs/merchant/checkout-terms.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	setCheckbox,
+	unsetCheckbox,
+} from '@woocommerce/e2e-utils';
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopper,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+import {
+	BILLING_DETAILS,
+	SIMPLE_VIRTUAL_PRODUCT_NAME,
+} from '../../../utils/constants';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `Skipping checkout tests`, () => {} );
+}
+
+describe( 'Merchant → Checkout → Can adjust T&S and Privacy Policy options', () => {
+	beforeAll( async () => {
+		await shopper.goToShop();
+		await page.goto( 'http://localhost:8889/?setup_terms_and_privacy' );
+		await expect( page ).toMatch( 'Terms & Privacy pages set up.' );
+		await shopper.block.emptyCart();
```

Comment:
```suggestion
		//Create a new page with a title: Terms and Conditions page
		await createNewPost( {
			postType: 'page',
			title: 'Terms and Conditions page',
		} );
		await saveOrPublish();
		//Navigate to WooCommerce > Settings > Advanced  to set up 'Terms and Conditions page'
		await merchant.openSettings( 'advanced' );
```

The above code will create a new page and browse to WooCommerce > Settings > Advanced tab, where we can choose our page and save the settings.


Similarly, the privacy policy page is already present. We just need to save it on the `/wp-admin/options-privacy.php` page.

With the above method, we eliminate the dependency on an external plugin. Please correct me if my approach is incorrect.

Reply:
See previous comment. 😀